artifacts/images/agent-build.Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the proxy-agent binary
+FROM golang:1.12.1 as builder
+
+# Copy in the go src
+WORKDIR /go/src/sigs.k8s.io/apiserver-network-proxy
+COPY pkg/    pkg/
+COPY cmd/    cmd/
+COPY proto/  proto/
+COPY vendor/ vendor/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o proxy-agent sigs.k8s.io/apiserver-network-proxy/cmd/agent
+
+# Copy the loader into a thin image
+FROM ubuntu:latest
```

Comment:
I think these should work with `scratch` right?

Reply:
Give it a try. At some point we may want an image with more tools in it.